new_platforms/nuget/build/native/openenclave.targets

```diff
@@ -319,6 +362,27 @@
         </Link>
     </ItemDefinitionGroup>
 
+    <!-- Additional Dependencies for Hosts using the master branch -->
+    <ItemDefinitionGroup Condition="'$(OEType)' != 'Enclave' And $(OEIsMaster) == True">
+        <Link>
+            <!-- *-SGX-* -->
+            <LibraryDependencies Condition="$(OEIsSgx) == True">$(OESgxLibsU);%(LibraryDependencies)</LibraryDependencies>
+
+            <!-- arm-TZ-OPTEE-* -->
+            <LibraryDependencies Condition="$(OEIsTz) == True And $(OEIsARM) == True And $(OEIsNewPlatforms) == True">$(OETzHwArm)\oehost.lib;$(OETzHwArm)\oesocket_host.lib;$(OETzHwArm)\oestdio_host.lib;$(OETzHwArm)\mbedcrypto.lib;$(OETzHwArm)\mbedx509.lib;onecore.lib;rpcrt4.lib;%(LibraryDependencies)</LibraryDependencies>
+
+            <!-- x86-OPTEE-Simulation-Debug -->
+            <LibraryDependencies Condition="$(OEIsTz) == True And $(OEIsX86) == True And $(OEIsSim) == True And $(OEIsNewPlatforms) == True">$(OEOpteeSimX86)\oehost.lib;$(OEOpteeSimX86)\oesocket_host.lib;$(OEOpteeSimX86)\oestdio_host.lib;$(OEOpteeSimX86)\oehost_opteesim.lib;$(OEOpteeSimX86)\mbedcrypto.lib;$(OEOpteeSimX86)\mbedx509.lib;onecore.lib;rpcrt4.lib;%(LibraryDependencies)</LibraryDependencies>
+        </Link>
+    </ItemDefinitionGroup>
+    <ItemDefinitionGroup Condition="'$(OEType)' == 'Enclave' And $(OEIsMaster) == True">
+        <Link>
+            <IgnoreAllDefaultLibraries>true</IgnoreAllDefaultLibraries>
+            <AdditionalOptions>-nostdlib -nodefaultlibs -nostartfiles -Wl,--no-undefined -Wl,-Bstatic -Wl,-Bsymbolic -Wl,--export-dynamic -Wl,-pie -Wl,--build-id</AdditionalOptions>
+            <LibraryDependencies>oeenclave;mbedx509;mbedcrypto;oelibcxx;oelibc;oecore;%(LibraryDependencies)</LibraryDependencies>
+        </Link>
+    </ItemDefinitionGroup>
```

Comment:
Lines 380..382 apply to enclaves, so they should be part of the `<Link`> block starting at line 321, guarded by `$(OEIsMaster) == True`.

Reply:
Done